echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/api/Notification.groovy

```diff
@@ -41,7 +41,8 @@ class Notification {
     PAGER_DUTY,
     JIRA,
     BEARYCHAT,
-    GOOGLECHAT
+    GOOGLECHAT,
+    GITHUB
```

Comment:
Is `GITHUB` too generic of a notification type?

Would we ever want to do something other than set a status check? (add a comment?, etc.)

On the flip-side, it seems possible to use `GITHUB` and a secondary discriminator to differentiate on what action wants to be taken?